packages/stockflux-launcher/src/titlebar/Titlebar.css

```diff
@@ -0,0 +1,76 @@
+.titlebar {
+    background-color: var(--titlebar-color);
+    border: none;
+    display: inline-block;
+    height: 100%;
+    width: 100%;
+    -webkit-app-region: drag;
+}
+
+.column {
+    border: none;
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+}
+
+.add-padding-left {
+    padding-left: 24px;
+}
+
+.column-left {
+    float: left;
+    -webkit-app-region: no-drag;
+}
+
+.add-padding-right {
+    padding-right: 12px !important;
+}
+
+.column-right {
+    float: right;
+    -webkit-app-region: no-drag;
+}
+
+.title {
+    color: #FFFFFF;
+    font-size: 21px;
+    user-select:none;
+}
+
+.title-light {
+    font-family: 'Roboto';
+}
+
+.title-strong {
+    font-family: 'Roboto';
+    font-weight: 500;
```

Comment:
It should not be hardcoded, we have font-weights defined in `packages\stockflux-components\src\index.css`